inspectit-oce-demo/docker-compose-elastic.yml

```diff
@@ -0,0 +1,166 @@
+version: '2'
+services:
+  config-server:
+    image: openapm/spring-petclinic-config-server:inspectit-oce-demo
+    container_name: config-server
+    environment:
+     - INSPECTIT_SERVICE_NAME=config-server
+     - INSPECTIT_CONFIG_FILE_BASED_PATH=/agentconfig
+     - INSPECTIT_EXPORTERS_METRICS_PROMETHEUS_PORT=9096
+    mem_limit: 512M
+    entrypoint: ["sh", "-c","
+      wget --no-check-certificate https://github.com/inspectIT/inspectit-oce/releases/download/${INSPECTIT_OCE_VERSION}/inspectit-oce-agent-${INSPECTIT_OCE_VERSION}.jar -O inspectit-oce-agent.jar
+      && java -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -Djava.security.egd=file:/dev/./urandom
+      -javaagent:/inspectit-oce-agent.jar
+      -jar /app.jar"]
+    volumes:
+     - ./agentconfig/config-server:/agentconfig
+    ports:
+     - 8888:8888
+     - 9096:9096
+
+  discovery-server:
+    image: openapm/spring-petclinic-discovery-server:inspectit-oce-demo
+    container_name: discovery-server
```

Comment:
all hostnames are defined by docker and are given unique UUID, this creates different hosts in system dashboard although it's actually the same host and it should be only one available as it runs on the same machine. Luckily we can solve this easily by adding the `hostname: whatever` property to the services that are monitored by the agent (so excluding the influx, elastic, etc). If we use the same hostname in all services, we end up with just one host in the dashboard. Should we make this change?

I tried this and it works:

```
  api-gateway:
    image: openapm/spring-petclinic-api-gateway:inspectit-oce-demo
    container_name: api-gateway
    hostname: inspectit-oce-demo-host
```